client/src/components/Common/CertificationItem/index.js

```diff
@@ -22,16 +24,16 @@ const CertificationItem = ({
   const renderLink = () => {
     if (discovered) {
       return (
-        <a href={url} className="image-wrapper" target="_blank" rel="noopener noreferrer">
-          <img className="image" src={image} alt="certificate" />
-        </a>
+        <Tilt className="tilt" options={{ max: 20, scale: 1.05 }}>
+          <a href={url} className="image-wrapper" target="_blank" rel="noopener noreferrer">
+            <img className="image" src={image} alt="certificate" />
+          </a>
+        </Tilt>
       );
     }
 
     return (
-      <div className="image-wrapper">
-        <img className="image" src={hiddenImage} alt="certificate" />
-      </div>
+      <img className="image" src={hiddenImage} alt="certificate" />
```

Comment:
Finally, I decided not to add any effect to the hidden item. That makes sense to me because anyway the item is not interactable, so why should it have any effect?